### DISCUSSION TITLE
UploadedFileInput widget

### DISCUSSION THREAD

**Discussion by lsmoker** [jrief/django-formset#129] [Q&A]
The Delete button on this widget does not remove the file (after you have dragged or chosen a file in the dropbox area) from the 'upload_temp' folder on disk. Is this expected behavior?

**Comment by jrief** [comment]
This is unavoidable, because the delete button does not call any endpoint.

I have to implement a cleanup solution anyway which handles the following case:

Consider someone uploading a file, but then never submitting the form. The file now is kept in its temporary location but since that someone did not perform a submission it keeps dangling there. The only solution to this is to run a cleanup job in reasonable intervals and remove those dangling files.
